src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx

```diff
@@ -0,0 +1,94 @@
+</figcaption>
+
+## Install the infrastructure agent [#infra]
+
+To get your data into New Relic, install our infrastructure agent. Our infrastructure agent collects and ingests data from your app so you can keep track of your app’s performance.
+
+You can install the infrastructure agent two different ways: 
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview). 
+* If you’d rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/). 
+
+## Export Nextcloud data [#export]
+
+To start the integration process, you need to export your Nextcloud metrics with `nextcloud-exporter`. This eventually allows our Prometheus OpenMetrics integration to ingest data into New Relic. To export your Nextcloud data: 
+
+1. [Install Go](https://go.dev/doc/install) if you haven't already.
+2. Run the following Docker commands to access `nextcloud-exporter`:
+
+```
+docker pull ghcr.io/xperimental/nextcloud-exporter:latest
+docker pull xperimental/nextcloud-exporter:latest
+```
+
+3. Clone `nextcloud-exporter`:
+
+```
+git clone https://github.com/xperimental/nextcloud-exporter.git
+cd nextcloud-exporter
+```
+
+4. Create a binary file in the `nextcloud-exporter` folder by using `make`. and Name the file `nextcloud-exporter`.
+5. Complete token authentication. You need to be using Nextcloud 22 or later.
+
+```
+# Generate random value using OpenSSL.
+TOKEN=$(openssl rand -hex 32)
+# Set token using the occ command by going to "/var/www/nextcloud”
+occ config:app:set serverinfo token --value "$TOKEN"
+``` 
+6. From your cloned Nextcloud folder, run the following command to initiate username and password authentication:
+
+```
+./nextcloud-exporter --login <USERNAME> <PASSWORD> --server http://127.0.0.1
+```
+7. To access metrics for Nextcloud, run:
+
+```
+./nextcloud-exporter -u <YOUR_USERNAME> -p <YOUR_PASSWORD> --server http://127.0.0.1:9205
+```
+
+8. From your browser, go to `http:127.0.0.1:9205/metrics` to view your metrics.  
+
+## Integrate Prometheus OpenMetrics with Nextcloud [#prometheus]
+
+With your Nextcloud metrics exposed, you can use our Prometheus OpenMetrics integration to scrape data.
+
```

Comment:
Install our [Prometheus OpenMetrics integration](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-update-or-uninstall-your-prometheus-openmetrics-integration/)

You need to configure the nri-prometheus-config.yml file. Your configuration file should match our snippet in the [nri-prometheus](https://github.com/newrelic/nri-prometheus/blob/main/configs/nri-prometheus-config.yml.sample) repository